blazor/pdfviewer/how-to/load-office-files.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: Load office files in Blazor PDF Viewer | Syncfusion
+description: Learn here all about how to load the microsoft office files like powerpoint in the Syncfusion Blazor PDF Viewer component and more.
+platform: Blazor
+control: PDF Viewer
+documentation: ug
+---
+
+# Load Microsoft Office files in Blazor PDF Viewer Component
+
+The PDF Viewer library allows you to load Microsoft office files such as powerpoint, excel, word and image using the success event of uploader.
+
+In the below code, word document is converted in to pdf document and return that pdf document into base64 string which then loaded in to the PDF Viewer. Similarly, loaded the powerpoint, excel and image in to the PDF Viewer.
```

Comment:
In the following code, a word document is converted into a PDF document and returned that PDF document into base64 string, which is then loaded into the PDF Viewer. Similarly, load the PowerPoint, Excel, and image into the PDF Viewer.